src/hal/src/pso/graphics.rs

```diff
@@ -126,11 +157,13 @@ pub struct Rasterizer {
     pub cull_face: Option<CullFace>,
     /// Which vertex winding is considered to be the front face for culling.
     pub front_face: FrontFace,
-    ///
+    /// Whether or not to enable depth clamping; when enabled, instead of
+    /// fragments being omitted when they are outside the bounds of the z-plane,
+    /// they will be clamped to the min or max z value.
     pub depth_clamping: bool,
-    ///
+    /// What depth bias, if any, to use for the drawn primitives.
     pub depth_bias: Option<DepthBias>,
-    ///
+    /// DOC TODO: Not sure what this is intended for, seems unused.
```

Comment:
dx12 only feature, controls how triangles will be rasterized depending on their overlap with pixels
currently not implemented afaik